website/docs/terminology.md

```diff
@@ -230,6 +235,8 @@ By repeating this compression, an arbitrary number of related [receipts] (e.g. a
 
 ### Recursion Circuit
 
+[Recursion Circuit]: #recursion-circuit
```

Comment:
Would you mind putting all references at the bottom of the file? This makes it a lot easier to maintain over time since we can find all the links in a common place in all files.

Reply:
If it's ok with you, let's land this as is and do that on a follow-up PR. There are a ton of other links on this page and I think implementing that change on this PR will distract from the more substantive changes being implemented here.